### ISSUE TITLE
Question: `vue/html-end-tags`

### ISSUE THREAD

**Issue by armano2** [vuejs/eslint-plugin-vue#160]
**Tell us about your environment**

* **ESLint Version:** 4.4.1
* **eslint-plugin-vue Version:** 3.11.0
* **Node Version:** 8.3.0

**Question**

For now rule `vue/html-end-tags` works only for non void html tags as described in doc: 
https://github.com/vuejs/eslint-plugin-vue/blob/master/docs/rules/html-end-tags.md#todo-brbr
```
TODO: <br></br>

parse5 does not recognize the illegal end tags of void elements.
```

We are not longer use `parse5` but issue is stil present with new parser.

**Comment by armano2** [comment]
@mysticatea are you planing to add support for it?

**Comment by mysticatea** [comment]
Thank you for this issue.

Yeah, this is one that we should solve until the official release.
The current state is:

- Illegal end tags are syntax errors, so those are caught by `no-parsing-error` rule.
- Vue.js generates errors if people omit end tags since [v2.2.0](https://github.com/vuejs/vue/releases/tag/v2.2.0) even if those are omittable in HTML.  
  (so it does not require the option which enforces to omit omittable end tags.)

So I think we should just update the document.
I will do after I get done for indent rules.
